R/SoilProfileCollection-integrity.R

```diff
@@ -139,7 +140,7 @@ setMethod('reorderHorizons',
                                    order(as.character(h[[idname(object)]]),
                                          h[[horizonDepths(object)[1]]]))
             
-            h <- aqp:::.as.data.frame.aqp(h[current.order,], 
+            h <- .as.data.frame.aqp(h[current.order,], 
```

Comment:
Is this going to be necessary for all functions that modify `@horizons` or slot previously containing a `data.frame`?

Reply:
Yes, if you want to be sure that you preserve the datatype the user set. It incurs no unnecessary overhead unless something occurred that changed the type.